flask_graphql/graphqlview.py

```diff
@@ -142,10 +143,10 @@ def get_response(self, request, data, show_graphiql=False):
     def json_encode(self, request, d, show_graphiql=False):
         pretty = self.pretty or show_graphiql or request.args.get('pretty')
         if not pretty:
-            return json.dumps(d, separators=(',', ':'))
+            return json.dumps(d, separators=(',', ':'), cls=self.json_encoder)
```

Comment:
Any reason to continue using `json.dumps` here rather than the encoder directly?
```py
return self.json_encoder(separators=(',' ':')).encode(d)
```

I believe they're functionally equivalent but the `.encode()` route reduces the dependency on the builtin `json` module, since there are several json libraries that could be reasonably used: http://artem.krylysov.com/blog/2015/09/29/benchmark-python-json-libraries/